components/ResultListPage/MainMenu/index.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function MainMenu({ currentRound }) {
+  if (currentRound === 3) {
```

Comment:
you may want to consider this to be based on the array of questions you have and take the final length (-1) as the condition. In this case the 3 is hardcoded and is not variable in case you want to add or remove questions.

Reply:
I only added the if statement because it doesn't look good visually if you see it every round, there it will not be necessary to add questions
thanks!